OpenKJ/OpenKJ.pro

```diff
@@ -188,6 +188,7 @@ SOURCES += main.cpp\
     karaokefileinfo.cpp \
     dlgeditsong.cpp \
     soundfxbutton.cpp \
+    runguard/runguard.cpp
```

Comment:
This line is missing a trailing \
This is required to tell qmake that the list of files continues on the next line